Makefile

```diff
@@ -69,6 +73,11 @@ build/.static: $(shell find hugo -type f)
 	@echo "INFO     Creating and populating build directory..."
 	@mkdir -p build
 	@cp -r hugo/* build
+	@echo >> build/config.toml
+	@echo "[params]" >> build/config.toml
```

Comment:
I'm not sure if that would break if we ever added a `[params]` section to `hugo/config.toml`, but I'm happy to put that concern aside for now.